src/sidebar/NavItem.tsx

```diff
@@ -65,6 +65,7 @@ export function NavItem({ node, subItem = false }: NavItemProps) {
 
   const handleNavigation = (isNavigatingForward?: boolean) => {
     if (node.dataSources) {
+      const [jsonataLink, jsonataError] = jsonata(node.externalUrl || '');
```

Comment:
This change is not working, I'm getting an errors on the examples
![image](https://github.com/user-attachments/assets/30a822fb-a48d-4f08-85d8-16f287090ea9)